libs/@guardian/source-foundations/src/typography/font-styles.ts

```diff
@@ -130,16 +130,8 @@ export const fontStyleToStringFunction =
 			font-family: ${fontFamily};
 			font-size: ${typeof fontSize === 'number' ? `${fontSize}px` : fontSize};
 			line-height: ${lineHeight};
-			${fontWeight ? `font-weight: ${fontWeight}` : ''};
-			${fontStyle ? `font-style: ${fontStyle}` : ''};
-
-			/*
-			 * Child elements (e.g. <Link/>) can use this variable
-			 * to set the thickness of their underline.
-			 *
-			 * The thickness for each font type and weight is defined
-			 * in the underlineThickness object.
-			 */
```

Comment:
did you mean to remove this comment?

Reply:
Yes, as there doesn't seem to be any need to output this comment every time the typography API is called. There's a comment in the source code for this function that explains its purpose which seems sufficient?

Reply:
oh apologies, i missed that, yes agree